### PR TITLE
Fix the elements check in the `sectionwizard.js` script

### PR DESCRIPTION
**Pull request by qzminski**
Exactly the same problem with section wizard as it was with module wizard, see #7017

If that one got merged, I hope we can get this one through as well 🙂

### CODE REVIEW

File: core-bundle/public/backend.be7cab7b.js.LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+/*! regenerator-runtime -- Copyright (c) 2014-present, Facebook, Inc. -- license (MIT): https://github.com/facebook/regenerator/blob/main/LICENSE */
```
Comment (ausi) [inline core-bundle/public/backend.be7cab7b.js.LICENSE.txt]: This seems to be unrelated to this pull request.

**Review by ausi — DISMISSED** [review]
LGTM, apart from the unrelated change.

File: core-bundle/public/backend.be7cab7b.js.LICENSE.txt (hunk 1 above)
Comment (qzminski) [reply core-bundle/public/backend.be7cab7b.js.LICENSE.txt]: Woopsy, no idea how it did get there… I'll remove it.

**Review by ausi — DISMISSED** [review]

**Review by leofeyer — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @qzminski.
